tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_proactive_IPI.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import pytest
+import random
+from ocs_ci.framework.testlib import (
+    tier4, ManageTest, aws_platform_required, ignore_leftovers)
+from ocs_ci.ocs import machine
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs.resources import pod
+from tests.helpers import get_worker_nodes
+from tests.helpers import wait_for_resource_state
+from tests.sanity_helpers import Sanity
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@ignore_leftovers
```

Comment:
what is the leftover you are expecting to have?

Reply:
After the new node is added and the pods move on the new node, we will see a new drain pod on the node as below,
"rook-ceph-drain-canary-0f7dd5f7b4476f0f437e6748306a9377-94bfkpv".
Since this one is newly run, it is failing with resource left over exception. 
So, to ignore that used "ignore_leftovers".